book-1-orientation/chapters/INSTALLATIONS.md

```diff
@@ -139,6 +156,98 @@ If you haven't already set up Visual Studio Code to format your code and autosav
 
 [SQL Server Express](https://www.microsoft.com/en-us/sql-server/sql-server-downloads). This is the database tool where all your data will be stored for the server-side course. Click the Download button under the Express option. During the setup, take the basic installation option
 
+
+## Installation Check
+Now that you have finished the installation process, please double check that you have not missed something.
+
+### Installation Check: Start Menu
+Check that you have the following programs by pressing your Windows key (Windows keyboard) or Command key (Apple keyboard) to open the Start menu, and begin typing in the name of the following progams to see that they are installed:
+
+```ssh
+Zoom
+Slack
+Postman
+Google Chrome
+Visual Studio
+Visual Studio Code
+Windows Terminal
+Mozilla Firefox (if you installed it)
+SQL Server 2019 Configuration Manager
+```
+
+### Installation Check: Terminal Commands
+Finally, run the following commands without quotes in Windows Terminal to check that the following are installed or set correctly:
+
+```ssh
```

Comment:
These items should look something like this

```sh
$ git --version
git version 2.28.0.windows.1
```

Reply:
Andy, I notice that a number of the earlier terminal commands are missing the $. Should I go back and add them in? An example would be line 77.

```sh
$ git config --global init.defaultBranch main
```